scrapper.py

```diff
@@ -44,20 +77,111 @@ def get_search_urls(self):
         pass
```

Comment:
This method is supposed to return seed_urls value, as stated in the docstring

Reply:
now it does